application/src/js/cms_autosave/click_to_edit.js

```diff
@@ -66,32 +70,59 @@
 
         function saveAndPreview(event) {
 
+            var displayText;
+
             /* TODO: MAKE CALL TO SAVE THE UPDATED FIELD */
             if (text_inputs[0]) {
-                var value = text_inputs[0].value.trim();
-                if (value == '') {
-                    preview.classList.add('empty');
-                    preview.textContent = preview.getAttribute('data-empty-text');
-                } else {
-                    preview.textContent = value;
-                    preview.classList.remove('empty');
+                displayText = text_inputs[0].value.trim();
+            }
+            else if (text_area) {
+                displayText = text_area.value.trim();
+
+                if (text_area.classList.contains('for-bullets')) {
+                    preview.innerHTML = displayText.replace(/^\*\s/gm, "<li>").replace(/\n/g, '</li>');
+                } else if (text_area.classList.contains('for-paragraphs')) {
+                    preview.innerHTML = displayText.replace(/\n/g, "<br>").replace(/\n/g, '</li>');
                 }
             }
-            if (text_area) {
-                var value = text_area.value.trim();
-                if (value == '') {
-                    preview.classList.add('empty');
-                    preview.textContent = preview.getAttribute('data-empty-text');
-                } else {
-                    if (text_area.classList.contains('for-bullets')) {
-                        preview.innerHTML = value.replace(/^\*\s/gm, "<li>").replace(/\n/g, '</li>');
-                    } else if (text_area.classList.contains('for-paragraphs')) {
-                        preview.innerHTML = value.replace(/\n/g, "<br>").replace(/\n/g, '</li>');
-                    } else {
-                        preview.innerHTML = value;
+
+            else if (checkboxesAndRadios.length > 0) {
+
+                var checkedLabels = []
+
+                for (var i = 0; i < checkboxesAndRadios.length; i++) {
+
+                    if (checkboxesAndRadios[i].checked) {
```

Comment:
This works for checkboxes, but doesn't seem to be working for radios - "Publication frequency" still just gets the `empty_message` when "save" is clicked.